docs/resources/workspace_mod_variable.md

```diff
@@ -57,6 +57,7 @@ resource "steampipecloud_workspace_mod_variable" "mandatory_tags" {
 
 The following arguments are supported:
 
+- `organization` - (Optional) The handle of the organization if the workspace belongs to an org.
```

Comment:
Can we please update this list to have the required arguments first and then place the optional ones in sorted order?